examples/gno.land/p/demo/mux/query.gno

```diff
@@ -0,0 +1,51 @@
+package mux
+
+import "strings"
+
+type UrlQuery map[string][]string
+
+/*
```

Comment:
I used this code for a similar thing in #2554. Of course the order in a map is unordered. But at a first glance it's quite similar. Note it's not yet been reviewed. I have been interested in querystring myself lately, so following this!

```
func BreakToMap(querystring string) (map[string]string, error) {
	m := make(map[string]string)
	for _, s := range strings.Split(querystring, "&") {
		if len(strings.TrimSpace(s)) == 0 {
			continue
		}
		idx := strings.Index(s, "=")
		if idx > -1 {
			m[s[:idx]] = s[idx+1:]
		} else {
			m[s] = ""
		}
	}
	return m, nil
}
```